lib/libzrepl/data_conn.c

```diff
@@ -380,6 +377,9 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 	while (1) {
 
 		if (ZVOL_IS_REBUILDING_FAILED(zinfo->zv)) {
+			LOG_ERR("rebuilding already failed.. for %s..",
```

Comment:
Since we added new state to rebuild, we need to change macro ZVOL_IS_REBUILDING_FAILED to check if state is set to ZVOL_REBUILDING_ERRORED
to error out rebuild operation on all other peers.

Reply:
good find sir. Fixed the same in uzfs_write_data also. Thanks.